chain/src/engine.rs

```diff
@@ -99,24 +109,17 @@ impl<B: Blob, E: Clock + GClock + Rng + CryptoRng + Spawner + Storage<B> + Metri
         .await;
 
         // Create the consensus engine
-        let journal = Journal::init(
```

Comment:
`threshold-simplex` now handles the creation and use of `journal` (as it now requires some `Codec` type internal to `consensus`)